src/turbomind/kernels/gemm/test/test_utils.cu

```diff
@@ -84,7 +84,7 @@ FastCompare(const T* src, const T* ref, int dims, int bsz, cudaStream_t stream,
         thrust::cuda::par.on(stream),
         zip_iter,
         zip_iter + count,
-        [=] __device__(auto tup) {
+        [=] __host__ __device__(thrust::tuple<float, float> tup) -> Tuple {
```

Comment:
May we also update with the turbomind repo https://github.com/InternLM/turbomind